gsl.copy_modules.sh

```diff
@@ -72,14 +72,16 @@ function generate_copy_modules(path_prefix)
 
     emit_initialize()
 
-   for generate.repository by name as _repository
-       echo(" Evaluating repository: $(_repository.name)")
-        emit_repository_initialize(_repository, my.path_prefix)
+    for generate.repository by name as _repository
+        echo(" Evaluating repository: $(_repository.name)")
+        define modulesTargetPath = \
+            "$(my.path_prefix)/$(_repository.name)/builds/cmake/modules/"
```

Comment:
It is not clear why the full path has been externalized given that only the prefix and repository vary.

Reply:
Instead of generating the same output address twice, I only generate it once. The advantage is simpler maintenance, removal of unnecessary duplication and fewer parameters.

Reply:
It could be considered an abstraction leak, unnecessarily complicating the outer loop by constructing a path within the loop as it is of no consequence to the loop. It could also be considered inconsistent with the approach taken generally in this repository. @pmienk any comment?

Reply:
The pattern of avoiding that leak is followed throughout much of the gsl for strings utilizing path/repository due to the common necessity of those properties for much of the generation. In this case it could go either way, but other files complications make it desirable to avoid the bloat of these kind of optimizations.

Reply:
I would like to accept the `cd \$(dirname "$0")` and not this one. I can make the change if you are not set up for it @ldz1 .

Reply:
Ah, I see that this was a spacing change only to a previous change.

Reply:
@evoskuil After all, I corrected it according to your intentions. I forgot to inform you. Anyway, the matter is already settled. However... Evoskuil, if you still have the opportunity to add my change, it is worth it, because there are still some spacing improvements.

Reply:
Sorry, I got a little mixed up merging several changes. I’ll try to find these.